pkg/util/strings/strings.go

```diff
@@ -91,3 +91,9 @@ func GetOnlyNumbers(yourString string) string {
 func GetSignedNumbers(yourString string) string {
 	return getOnlyMatch(signedNumbersMatch, yourString)
 }
+
+// Split the string into two trimmed strings based on the passed separator
+func SplitKeyValue(s string, separator string) (string, string) {
```

Comment:
Do you need this? There is also splitAndTrim in addon_meta.

Reply:
`splitAndTrim` doesn't accept another parameter for `separator`. I will move this into strings package as splitKeyAndValue has been used in other places as well.